target/linux/ramips/dts/YOUKU-YK2.dts

```diff
@@ -0,0 +1,151 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "youku,yk-l2", "mediatek,mt7621-soc";
+	model = "YOUKU YK2";
```

Comment:
Model name should be Youku YK-L2 instead.

Reply:
ok , will change.